flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/TableEnvironment.java

```diff
@@ -175,7 +175,7 @@
 	 * @return The resulting {@link Table}.
 	 * @throws ValidationException if no table is found using the given table path.
```

Comment:
I don't know if we should document these exceptions. Usually, there is a lot that can go wrong under the hood.